xformers/components/residual.py

```diff
@@ -34,21 +26,37 @@ class LayerNormStyle(str, Enum):
     Post = "post"
 
 
+class RequiresWrappedInputs:
```

Comment:
classes which derive from this class only accept a single input list (makes it impossible to subtly footgun)